.github/workflows/docs-style.yml

```
@@ -0,0 +1,40 @@
+name: Documentation style
+on:
+  push:
+    branches:
+      - master
+      - 0.x
+      - 1.x
+  pull_request:
+    types: [opened, reopened, synchronize, labeled, unlabeled, ready_for_review]
+jobs:
+  releasenotes:
+    name: Generate and check release notes
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.8'
+
+      - name: Install Dependencies
+        run: pip install riot
+
+      - name: Lint release notes
+        run: riot run reno lint
```

```suggestion
        run: riot run -s reno lint
```

or do we need to install the package?